cmsmon-tools.spec

```diff
@@ -0,0 +1,86 @@
+### RPM cms cmsmon-tools 0.3.10
+## NOCOMPILER
+
+%define pkg CMSMonitoring
+%define ver %realversion
+%define cmsmon_commands nats-sub nats-pub nats-exitcodes-termui dbs_vm
+%define flags -ldflags="-s -w -extldflags -static" -p %{compiling_processes}
+Source0: https://github.com/dmwm/%pkg/archive/%ver.tar.gz
+
+BuildRequires: go
+
+# RPM macros documentation
+# http://www.rpm.org/max-rpm/s1-rpm-inside-macros.html
+%prep
+%setup -D -T -b 0 -n %pkg-%ver
+
+%build
+cd ..
+cd %pkg-%ver
+echo "build $PWD"
+ls
+mkdir -p gopath/bin
+export GOPATH=$PWD/gopath
+export GOCACHE=%{_builddir}/gocache
+go get github.com/dmwm/cmsauth
+go get github.com/vkuznet/x509proxy
+go get github.com/sirupsen/logrus
+go get github.com/prometheus/client_golang/prometheus
+go get github.com/prometheus/common/log
+go get github.com/prometheus/common/version
+go get github.com/shirou/gopsutil/cpu
+go get github.com/shirou/gopsutil/mem
+go get github.com/shirou/gopsutil/load
+go get github.com/shirou/gopsutil/process
+go get github.com/go-stomp/stomp
+go get github.com/nats-io/nats.go
+go get github.com/gizak/termui/v3
+
+# build monit tools
+cd src/go/MONIT
+go build %flags monit.go
+cd -
+# build NATS tools
+cd src/go/NATS
+for cmd in %cmsmon_commands; do
+if [ -f $cmd.go ]; then
```

Comment:
by the way, why this check? Shouldn't we know what command we have to build and if they are missing then we should fail ... right?

Reply:
because cmsmon_commands lists scripts from different areas, e.g. monit.go comes from src/go/MONIT area, while nats-sub.go comes from src/go/NATS. This is why I check that actual file we intend to compile exists. I think we should keep this structure if we want to use  common cmsmon_commands which lists names from different areas of our repository.

Reply:
if you agree with my comment here, I think everything else is ready and can be merged.